### DISCUSSION TITLE
Enable Localization option missing with Microsoft Search data source, Tags show GUID in Filters

### DISCUSSION THREAD

**Discussion by mw-new-test** [microsoft-search/pnp-modern-search#3245] [Ideas]
**Version used**
Ex: 4.9.1

**Describe the bug**
When ows_taxId_MetadataAllTagsInfo is mapped to a RefinableStringXX, and used as a Filter, it shows values as L0|#xxxxxxxxx-xxx-xxx-xxx-xxxxxxxxxx|value
Data source used - Microsoft Search

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Configure PnP Modern search results webpart and filters webparts and connect them
2. Select Microsoft Search as Data source
3. Add owstaxidmetadataalltagsinfo and RefinableStringXX to Selected properties
4. Please note Enable localization configuration is missing
5. Add RefinableStringXX to Filters webpart
6. See error

**Expected behavior**
Tag Filter values should only show the name instead of L0|#xxxxxxxxx-xxx-xxx-xxx-xxxxxxxxxx|value

**Screenshots**

**Desktop (please complete the following information):**
 Chrome/Edge

**Additional context**

**Comment by wobba** [comment]
This is not a bug, it's not implemented.
